style.css

```diff
@@ -4,7 +4,7 @@ Theme URI: https://easydigitaldownloads.com/downloads/vendd
 Author: EDD Team
```

Comment:
Should we go ahead and update the author and author URI here since we have been doing it for the plugins? Technically also in the constants in functions.php.

Reply:
Made an issue for that: https://github.com/easydigitaldownloads/vendd/issues/273